src/python/nimbusml/pipeline.py

```diff
@@ -2581,6 +2581,10 @@ def summary(self, verbose=0, **params):
             self._run_time = time.time() - start_time
             raise e
 
+        # .summary() not supported if size of summary_data
+        # is less or equal to 1 (if only PredictedName in summary_data)
+        if summary_data.size == 1 and summary_data.columns.values == ["PredictorName"]:
+            raise TypeError("One or more transformers in this pipeline do not support the .summary() function.")
```

Comment:
Should "transformers" be "predictors" here?

Reply:
I was thinking they could be interchangeable in this case, but I believe you're right, "predictors" is more accurate here.